cinema/serializers.py

```
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Ensure that the `validate_ticket` method in the `Ticket` model is correctly implemented and handles the validation logic as expected. This method should raise a `ValidationError` if the ticket data is invalid. Double-check the method signature and logic in the `Ticket` model to confirm it matches the expected usage here.